components/finalMessage/finalMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+
+import { Typography, Button } from '@mui/material'
+
+import { StyledFinalMessageContent, StyledTextArea } from '../../pageStyles/emailSender.styles'
+
+type FinalMessageProps = {
+    id: string,
+    to: string,
+    subject: string,
+    content: string,
+    parentCallback: Function,
+};
+
+export default function FinalMessage
+({ id, to, subject, content, parentCallback }: FinalMessageProps) {
+  const toMessage = to
+  const subjectMessage = subject
+  const idMail = id
+
+  const [isEditing, setIsEditing] = useState(false)
+  const [messageContent, setMessageContent] = useState(content)
+
+  const handleEditButton = () => {
+    setIsEditing(true)
+  }
+
+  const handleEditMessage = (e: any) => {
+    setMessageContent(e.target.value)
+  }
+
+  const handleSubmitButton = () => {
+    setIsEditing(false)
+    console.log(idMail)
+    parentCallback(idMail, to, subject, messageContent)
+  }
+
+  return (
+        <div>
+            <Typography variant="body1">To:{toMessage}</Typography>
+            <Typography variant="body1">Subject: {subjectMessage}</Typography>
+            <br />
+            <Typography variant="body1">Content:</Typography>
+            <br />
+            <Typography variant="body1">
+
+                {isEditing
```

Comment:
I think we can improve the UX a bit if we move the edit / save button to the upper right hand corner. It's more of a standard spot for an editing button I feel like and will reduce the vertical space each printed message takes up! 

Something like this? 
<img width="1512" alt="Screen Shot 2022-07-15 at 3 31 49 PM" src="https://user-images.githubusercontent.com/59738880/179297941-776872f8-26d5-4b8b-a00b-7d8197dc988d.png">
 
We can also use the same button for both editing + saving. If you remove the buttons from below and add the following code below line 39, this handles both functions with the same button. To right align it we can style the button and use float: right

`<Button
        variant="contained"
        size="small"
        onClick={isEditing ? handleSubmitButton : handleEditButton}
      >
        {isEditing ? 'Save' : 'Edit'}
      </Button>`

Reply:
Ohh yes I agree with that!! Another option would be to put the edit button right under the 'Content' label so that it is clear that you can only edit the message. Or if you want to keep the edit underneath the message, I think it should have a little more space.